### PR TITLE
Added player position

### PR DESCRIPTION
**Pull request by SwedishSock**
Now the player's position changes if you do not play on a rig connected to a VR device, which helps you look around better since the camera won't be in the ground.

Also changed some other scripts to use the correct GetEntityByGUID function.

### CODE REVIEW

**Comment by NinjaDanz3r** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] AncientFungus:LGEPR: Cppcheck cannot find all the include files (use --check-config for details) [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AmissingInclude)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: Game/Resources/Constant Scene/Minecart/Script/Player.as [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    Player(Entity @entity){
+        @hub = Managers();
+        @self = @entity;
+        first = true;
+        // Remove this if updates are not desired.
+        RegisterUpdate();
+    }
+
+    // Called by the engine for each frame.
```
Comment (ludvigarlebrink) [inline Game/Resources/Constant Scene/Minecart/Script/Player.as]: Unnecessary comment, since this is true with all scripts.

Comment (SwedishSock) [reply Game/Resources/Constant Scene/Minecart/Script/Player.as]: This is not my doing, I will not acquiesce this change. The script stays as is!

**Comment by NinjaDanz3r** [comment]
iamheman

**Comment by NinjaDanz3r** [comment]
iamheman

**Pull request merged**